Examples/UIExplorer/UIExplorerUnitTests/RCTUnicodeDecodeTests.m

```diff
@@ -0,0 +1,59 @@
+/**
+ * The examples provided by Facebook are for non-commercial testing and
+ * evaluation purposes only.
+ *
+ * Facebook reserves all rights not expressly granted.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+ * OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON INFRINGEMENT. IN NO EVENT SHALL
+ * FACEBOOK BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN
+ * AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+@import XCTest;
+@import ObjectiveC;
+
+#import "RCTNetworking.h"
+
+@interface RCTNetworking ()
+
++ (NSString *)decodeTextData:(NSData *)data fromResponse:(NSURLResponse *)response withCarryData:(NSMutableData*)inputCarryData;
+
+@end
+
+@interface RCTUnicodeDecodeTests : XCTestCase @end
+
+@implementation RCTUnicodeDecodeTests
+
+- (void)testUnicodeDecode
+{
+  NSString* unicodeString = @"😀😀";
```

Comment:
for safer integration with source control, we usually don't put unicode character directly in the code. Instead we use `\u` representation, so I think this line can be written as:

`@"\uD83D\uDE00\uD83D\uDE00";`